l10n_co_res_partner/LICENSE

```diff
@@ -0,0 +1,661 @@
+GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Full licence is not necessary anymore.

You can use the link s in the readme that's enought